docs/docs/cmd/spo/file/file-sharinglink-set.md

```diff
@@ -0,0 +1,78 @@
+# spo file sharinglink set
+
+Updates a sharing link of a file
+
+## Usage
+
+```sh
+m365 spo file sharinglink set [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+:	The URL of the site where the file is located.
+
+`--fileUrl [fileUrl]`
+:	The server-relative (decoded) URL of the file. Specify either `fileUrl` or `fileId` but not both.
+
+`--fileId [fileId]`
+:	The UniqueId (GUID) of the file. Specify either `fileUrl` or `fileId` but not both.
+
+`--id <id>`
+: The ID of the sharing link.
+
+`--expirationDateTime <expirationDateTime>`
+:	The date and time to set the expiration. This should be defined as a valid ISO 8601 string. This options only works for anonymous links.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Updates an anonymous sharing link from a file with the expirationDateTime parameter
+
+```sh
+m365 spo file sharinglink set --webUrl https://contoso.sharepoint.com --fileUrl "/Shared Documents/Document.docx" --expirationDateTime "2023-01-09T16:57:00.000Z"
```

Comment:
Could we use a server relative URL here which is more commonly used?
Also, we strive to have at least 2 examples per command, can we provide another example with file ID for example?